plugins/source/aws/resources/services/wellarchitected/lenses.go

```diff
@@ -0,0 +1,88 @@
+package wellarchitected
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/wellarchitected"
+	"github.com/aws/aws-sdk-go-v2/service/wellarchitected/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v3/schema"
+	"github.com/cloudquery/plugin-sdk/v3/transformers"
+)
+
+// lens info consists of types.LensSummary & types.Lens fields
+type lens struct {
+	*types.LensSummary
+	// extra field from types.Lens
+	ShareInvitationId *string
+	// extra field from types.Lens
+	Tags map[string]string
+}
+
+func Lenses() *schema.Table {
+	name := "aws_wellarchitected_lenses"
+	return &schema.Table{
+		Name:        name,
+		Description: `https://docs.aws.amazon.com/wellarchitected/latest/APIReference/API_Lens.html`,
+		Transform: transformers.TransformWithStruct(new(lens),
+			transformers.WithPrimaryKeys("LensAlias"),
+			transformers.WithUnwrapAllEmbeddedStructs(),
+			transformers.WithNameTransformer(client.CreateTrimPrefixTransformer("lens_")),
+		),
+		Multiplex:           client.ServiceAccountRegionMultiplexer(name, "wellarchitected"),
+		Resolver:            fetchLenses,
+		PreResourceResolver: getLens,
+		Columns:             schema.ColumnList{client.DefaultAccountIDColumn(true), client.DefaultRegionColumn(true)},
```

Comment:
I don't think we need the Region to be a PK as the ARN/Alias seems to always have it

Reply:
alias for official is `serverless` for example, so no, these need to stay =(